database/config.js

```diff
@@ -1,22 +1,21 @@
 const mongoose = require("mongoose");
 
 const dbConection = async () => {
-    try {
-        await mongoose.connect(process.env.db_cnn, {
-            useNewUrlParser: true,
-            useUnifiedTopology: true,
-            useCreateIndex: true,
-            useFindAndModify: false,
-        });
-        // mongoose.set("useCreateIndex", true);
-        // mongoose.set("useFindAndModify", false);
-        console.log("Db online");
-    } catch (error) {
-        console.log(error);
-        throw new Error("Error a la hora de inicializar base de datos");
-    }
+  try {
+    await mongoose.connect(process.env.db_cnn, {
+      useNewUrlParser: true,
+      useUnifiedTopology: true,
+      useCreateIndex: true,
+      useFindAndModify: false,
+    });
+
+    console.log("Db online");
```

Comment:
Could you please add a comment before this line to explain why we decided to keep this console.log? I think it is neccesary